input/pagecontent/general-requirements.md

```diff
@@ -315,28 +145,39 @@ The full set of sub-elements is visible in the "Key Elements Table" or "Snapshot
 
 For example, the AU Core MedicationRequest Profile `note` element is labelled *Must Support* and has no *Must Support* sub-elements. When claiming conformance to this profile:
-- AU Core Responders **SHALL** be capable of providing a value in a `MedicationRequest.note` sub-element e.g. `MedicationRequest.note.text`.
-- AU Core Requestors **SHALL** be capable of processing the MedicationRequest resource with a value in `MedicationRequest.note`.
+- AU Core Responders **SHALL** correctly populate a value in any valid `MedicationRequest.note` sub-element if a value is known e.g. `MedicationRequest.note.text`.
+- AU Core Requesters **SHALL** accept the MedicationRequest resource without error if `MedicationRequest.note` is present and containing any valid sub-elements.
 
-For example, if AU Core Patient Profile name element is labelled *Must Support* and has *Must Support* sub-elements "family” and “given”. When claiming conformance to this profile:
-- AU Core Responders **SHALL** be capable of providing a value in `Patient.name.family` and `Patient.name.given`.
-- AU Core Requestors **SHALL** be capable of processing a Patient resource with a value in `Patient.name.family` and `Patient.name.given`.
+If a sub-element is labelled as *Must Support*: 
+- AU Core Responders **SHALL** correctly populate the element with all _Must Support_ sub-elements for which a value is known. 
+- AU Core Requesters **SHALL** accept resource without error if _Must Support_ sub-elements are present and containing any valid value.
+
+For example, in the AU Core Patient Profile, the ``name` element is labelled *Must Support* and has *Must Support* sub-elements `family` and `given`. When claiming conformance to this profile:
+- AU Core Responders **SHALL** correctly populate a value in `Patient.name.family` and `Patient.name.given` if the value for those sub-elements is known.
+- AU Core Requesters **SHALL** accept a Patient resource without error if `Patient.name` present and contains valid values in `family` and `given` sub-element.
 
 ##### Must Support - Resource References
-Some elements labelled as *Must Support* reference multiple resource types or profiles (e.g., Observation.performer). AU Core Responders **SHALL** support at least one referenced resource or profile data type for each element listed in the table below. AU Core Requesters **SHALL** support all referenced resources or profiles listed in the table below.
+Some elements labelled as *Must Support* reference multiple resource types or profiles such as `Observation.performer`. In such cases: 
+- AU Core Responders **SHALL** correctly populate the element with at least one referenced resource or allowed profile types if the value is known. 
+- AU Core Requesters **SHALL** accept resources without error if the element is present and containing any valid referenced resource or profiles.
```

Comment:
I find this very confusing - it just seems overly complicated for no reason. The first sentence already qualifies that some MS elements reference multiple resource types or profiles. 'valid reference' instead of 'valid referenced resource or profiles' is much clearer and does not take the attention away from the req for an AU Core Responder to correctly populate the element with at least one valid reference .. and AU Core Requester to accept the resource if the element is present and containing any valid reference.

Reply:
The pattern of leading sentence and bullet points follow the others and while I agree this is not very clear. The typo is "profile types" to be "profiles".